README.md

```diff
@@ -1,3 +1,43 @@
-AEMO GEL system icon package.
+# AEMO GEL Icon
+v1.1.1
 
-V1.1.1
\ No newline at end of file
+GEL icon library is a collection of web icons and system icons that are custom to AEMO. This section provides guidelines on how to use the current library and create additional icons when required.
+
+## Usage
+### Install
+Run the following command to install the package
+
+```bash
+npm install @danielwang/aemo-gel-icon@1.1.0
+```
+
+### Import 
+
+Then import the library with the following
+```js
+import '@danielwang/aemo-gel-icon/style.css'
+```
+
+### Use font icons by coding in HTML
+
+Gel icons use the `pi pi-{icon}` syntax such as pi pi-check following the reference from PrimeVue. A standalone icon can be displayed using an element like i or span.
+
+The icons are treated as fonts. So they can be resized by giving font-size styling to the icon class.
+
+```html
+<i class="pi pi-check" style="font-size: 2rem"></i>
+```
+
```

Comment:
please add steps of generating fonts files from svg by using the icomoon

Reply:
understood